### DISCUSSION TITLE
Backport [param] documentation tag to 3.x to make backporting class reference improvements more straightforward.

### DISCUSSION THREAD

**Discussion by Mickeon** [godotengine/godot-proposals#8754] [Engine Core]
Contributing to the class reference. 
I recently [backported the way the **Resource**'s class reference is written to to 3.x](https://github.com/godotengine/godot/pull/86642). I was lucky, because it wasn't too large. But, every use of the `[param]` tag needed to be carefully pruned from each method's description, and replace with a `[code]` tag, which can be a quite tedious process.

It would be a shame to end 3.x support without reliable documentation. So I'm wondering if there is interest in attempting to mitigate this process in some ways, this being one of them.

**Comment by Calinou** [comment]
I recall we dropped the idea of backporting https://github.com/godotengine/godot/pull/64008 to 3.x because it'd break all existing class reference translations.

**Reply by AThousandShips** [reply]
A half solution would be to make it opt-in, and skip the checks warning with `[code]my_param[/code]`, but feels a bit incomplete

**Reply by Mickeon** [reply]
I always assumed it would've been opt-in. The suggestion is to help backporting the class reference, not to add any features to `param`. A simple "The parser replacing `[param my_param]` with `[code]my_param[/code]`" sounds reasonable to me. If it can be done, that is.

**Reply by AThousandShips** [reply]
That would break the translations as above, or it's only available in English
